Source/WebCore/PAL/pal/PALSwift.h

```diff
@@ -28,8 +28,39 @@
 #include <cstdint>
 #include <wtf/Vector.h>
 
+namespace Cpp {
+
 using VectorUInt8 = WTF::Vector<uint8_t>;
 using SpanConstUInt8 = std::span<const uint8_t>;
+using OptionalVectorUInt8 = std::optional<WTF::Vector<uint8_t>>;
+
+
+// FIXME: remove when swift support is available rdar://118026392
+inline OptionalVectorUInt8 toOptionalVectorUInt8(VectorUInt8 val)
```

Comment:
WebKit naming convention here would be "makeOptional". We leave the type name out because it's in the argument signature. We use "make" to indicate constructing something, e.g. makeString().

Reply:
renamed.